ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml

```diff
@@ -5687,6 +5687,87 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build03
+  cron: 0 8 * * 3
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+    timeout: 8h0m0s
+  extra_refs:
+  - base_ref: main
+    org: openshift-qe
+    repo: ocp-qe-perfscale-ci
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-perfscale-qe
+    ci-operator.openshift.io/variant: rosa-hcp-4.14-stable-x86-loaded-upgrade-from-4.14
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-hcp-4.14-stable-x86-loaded-upgrade-from-4.14-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414
+  spec:
```

Comment:
Can you also add slack reporting after each periodic test?

```
reporter_config:
slack:
channel: '#ocp-qe-scale-ci-results'
job_states_to_report:
- success
- failure
- error
report_template: '{{if eq .Status.State "success"}} ✅ Job {{.Spec.Job}}
ended with {{.Status.State}}. <{{.Status.URL}}|View logs> ✅
{{else}} ⚠️ Job {{.Spec.Job}} ended with {{.Status.State}}. <{{.Status.URL}}|View
logs> ⚠️ {{end}}'
```

Reply:
Hi Paige, Thank you for review my code, I have added the code, thanks!